server/views/pages/event.njk

```diff
@@ -1,8 +1,14 @@
 {% extends 'layout/default.njk' %}
 
 {% block pageMeta %}
-  {% component 'open-graph', { pageConfig: pageConfig, article: event } %}
-  {% component 'twitter-card', { pageConfig: pageConfig, article: event } %}
+  {% set metaContent = {
+    type: 'wellcome:event' + '.' + event.format,
```

Comment:
Good question. Open Graph types to-date appear to be limited to music, video, article, book, profile, website.

> When the community agrees on the schema for a type, it is added to the list of global types. All other objects in the type system are CURIEs of the form
> 
> ```<head prefix="my_namespace: http://example.com/ns#">```
>``` <meta property="og:type" content="my_namespace:my_type" />```